apps/desktop-wallet/src/modals/ContactFormModal.tsx

```diff
@@ -73,12 +73,15 @@ const ContactFormModal = ({ contact, onClose }: ContactFormModalProps) => {
       dispatch(contactStoredInPersistentStorage({ ...contactData, id }))
       onClose()
 
-      posthog.capture(contactData.id ? 'Edited contact' : 'Saved new contact', {
-        contact_name_length: contactData.name.length
+      sendAnalytics({
+        event: contactData.id ? 'Edited contact' : 'Saved new contact',
+        props: {
+          contact_name_length: contactData.name.length
+        }
       })
-    } catch (e) {
-      dispatch(contactStorageFailed(getHumanReadableError(e, t('Could not save contact.'))))
-      posthog.capture('Error', { message: 'Could not save contact' })
+    } catch (error) {
+      dispatch(contactStorageFailed(getHumanReadableError(error, t('Could not save contact.'))))
+      sendAnalytics({ type: 'error', error, message: 'Could not save contact' })
```

Comment:
@mvaivre Do you think we should report the error here or not?

Reply:
I wondered, but I think it's ok.